docs/Testing.md

```diff
@@ -1,13 +1,129 @@
 <h1 align="center">Fastify</h1>
 
 ## Testing
+
 Testing is one of the most important parts of developing an application. Fastify is very flexible when it comes to testing and is compatible with most testing frameworks (such as [Tap](https://www.npmjs.com/package/tap), which is used in the examples below).
 
-<a name="inject"></a>
-### Testing with http injection
+Let's `cd` into a fresh directory called 'testing-example' and type `npm init -y` in our terminal.
+
+run `npm install fastify && npm install tap pino-pretty --save-dev`
+
+### Separating concerns makes testing easy
+
+ First we're going to separate our application code from our server code:
+
+app.js
+
+```js
+'use strict'
+
+const Fastify = require('fastify')
```

Comment:
I would only capitalize the variable name here if you intend to name the instance `fastify`, e.g.:

```js
const Fastify = require('fastify')
const fastify = Fastify(options)
```

Otherwise, this implies the function exported by the module is a constructor style function; it is not. The function exported by the module is a factory function.